msbuild-sonarqube-runner/tools/chocolateyinstall.ps1

```diff
@@ -1,8 +1,5 @@
-﻿$url = "https://github.com/SonarSource-VisualStudio/sonar-msbuild-runner/releases/download/1.1/MSBuild.SonarQube.Runner-1.1.zip"
-$checksum = "D9302247D6AAFAB2B76256BFC96FC8F74468B87A"
-
-Install-ChocolateyZipPackage 'msbuild-sonarqube-runner' `
-  -url "$url" `
-  -unzipLocation "$(Split-Path -parent $MyInvocation.MyCommand.Definition)" `
-  -checksumType 'sha1' `
-  -checksum "$checksum"
+﻿$packageName = 'msbuild-sonarqube-runner'
+$url = 'https://github.com/SonarSource-VisualStudio/sonar-msbuild-runner/releases/download/2.1/MSBuild.SonarQube.Runner-2.1.zip'
+$url64 = $url
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+Install-ChocolateyZipPackage $packageName $url $toolsDir $url64
```

Comment:
Can you add back the checksum? It's [highly recommended](https://github.com/chocolatey/choco/wiki/HelpersInstallChocolateyZipPackage#-checksum-string) to have it.